src/main/java/io/supertokens/webserver/api/oauth/OAuthAuthAPI.java

```diff
@@ -76,16 +78,16 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
                 }
             }
             response.add("cookies", jsonCookies);
-
+            response.addProperty("status", "OK");
             super.sendJsonResponse(200, response, resp);
 
         } catch (OAuthAuthException authException) {
 
             JsonObject errorResponse = new JsonObject();
             errorResponse.addProperty("error", authException.error);
-            errorResponse.addProperty("error_description", authException.errorDescription);
-
-            super.sendJsonResponse(400, errorResponse, resp);
+            errorResponse.addProperty("errorDescription", authException.errorDescription);
+            errorResponse.addProperty("status", "OAUTH2_AUTH_ERROR");
```

Comment:
The comment [here](https://github.com/supertokens/supertokens-core/issues/582#issuecomment-2265144074) says `AUTH_ERROR`, however in the code it's `OAUTH2_AUTH_ERROR` - which one is correct? Please modify the comment or the code.

Reply:
Sure, updated the comment. I think being more specific in the error status (OAUTH2_AUTH_ERROR instead of just (AUTH_ERROR) is a good thing.